change/@itwin-eslint-plugin-e19a7842-1ecb-4de7-b06d-ed44c1e4108b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
I am aware that this change might break some builds with existing no-internal rule turned on but this is technically new feature added to existing rule; hence I have set it to `minor`. Versioning semantics with eslint rules always confuse me so I am open to discussion on this one.